src/site-server.ts

```diff
@@ -189,4 +190,26 @@ export class SiteServer {
 			return { stdout: '', stderr: 'error when executing wp-cli command', exitCode: 1 };
 		}
 	}
+
+	async isSQLiteInstalled(): Promise< boolean > {
```

Comment:
Is this function checking if SQLite plugin is installed or if it is active? I think, based on the code inside the function, that it is to check if SQLite is active. If that is the case, for clarity maybe this can be renamed to `isSQLitePluginActivated` or something similar.

Reply:
Yep, good point. The function checks both that is installed and active. My rationale is that since the SQLite plugin is a must-use plugin if it's installed we also consider it active. However, I agree that it's more accurate to reference in the function's name that the plugin is active. I'll rename it.